Update rendering-to-html-report-builder-and-ssrs.md (#3675)
MicrosoftDocs/sql-docs: docs/reporting-services/report-builder/rendering-to-html-report-builder-and-ssrs.md

  
 Additional search and find functionality is provided by the ReportViewer Web Forms control.  
  

> ##  <a name="FontsOnClient"></a> Fonts on the Client Computer

Reviewer: ```suggestion
##  <a name="FontsOnClient"></a> Fonts on the client computer
```

>  When a custom font is used within the report, the computer that is used to view the report (the client computer) must have the custom font installed for the report to display correctly. If the font is not installed on the client computer, the report will diplay a system default font instead of the custom font.

Reviewer: ```suggestion
 When you use a custom font in a report, any computer that is used to view the report (the client computer) must have the custom font installed for the report to display correctly. If the font isn't installed on the client computer, the report displays a system default font instead of the custom font.
```
